### ISSUE TITLE
Unable to install adopt@1.11.0-0 (macOS)

### ISSUE THREAD

**Issue by pmonks** [shyiko/jabba#311]
**Steps to reproduce:**

1. Install Jabba using the direct download (I would have preferred to use homebrew, but wasn't sure of the latest status of issue #138), using the command: `curl -sL https://github.com/shyiko/jabba/raw/master/install.sh | bash; and . ~/.jabba/jabba.fish`
2. Install `adopt@1.11.0-0`, using the command: `jabba install adopt@1.11.0-0`

**Expected result:**
AdoptOpenJDK v11 is installed.

**Actual result:**
`java` binary for new JDK is not found - installation fails.

**Environment:**
Jabba v0.10.1
fish shell v2.7.1
macOS Mojave (v10.14)

**Notes:**

* I was able to manually download the GZIP file Jabba reports to be downloading (https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11%2B28/OpenJDK11-jdk_x64_mac_hotspot_11_28.tar.gz), and was able to extract it and locate the `java` binary in the sub-folder Jabba is checking, so I'm pretty certain this is not due to an issue with the GZIP file itself (unless Jabba is corrupting it during download somehow?  Seems unlikely...).
* I was able to install and use `openjdk@1.11.0`, so I believe Jabba itself is installed and configured correctly.

**typescript:**
```
>  curl -sL https://github.com/shyiko/jabba/raw/master/install.sh | bash; and . ~/.jabba/jabba.fish
Installing v0.10.1...

Skipped update of /Users/pmonks/.bashrc (source string already present)
Skipped update of /Users/pmonks/.bash_profile (source string already present)
Skipped update of /Users/pmonks/.zshrc (source string already present)
Adding source string to /Users/pmonks/.config/fish/config.fish

Installation completed
(if you have any problems please report them at https://github.com/shyiko/jabba/issues)
> 
```

&lt;shell session restarted here&gt;

```
> jabba ls-remote
1.11.0
1.10.0
1.10.0-2
1.8.181
1.6.65
adopt@1.11.0-0
adopt@1.10.0-2
adopt@1.10.0-1
adopt@1.9.0-4
adopt@1.9.0-0
adopt@1.8.181-13
graalvm@1.0.0-7
graalvm@1.0.0-6
graalvm@1.0.0-5
openjdk@1.12.0
openjdk@1.12.0-14
openjdk@1.11.0
openjdk@1.10.0
openjdk@1.10.0-2
openjdk@1.10.0-1
openjdk@1.9.0-4
zulu@1.11.0
zulu@1.10.0
zulu@1.10.0-2
zulu@1.10.0-1
zulu@1.9.0
zulu@1.9.0-7
zulu@1.9.0-4
zulu@1.9.0-1
zulu@1.9.0-0
zulu@1.8.181
zulu@1.8.172
zulu@1.8.163
zulu@1.8.162
zulu@1.8.152
zulu@1.8.144
zulu@1.8.131
zulu@1.8.121
zulu@1.8.92
zulu@1.8.91
zulu@1.8.72
zulu@1.8.71
zulu@1.7.191
zulu@1.7.181
zulu@1.7.171
zulu@1.7.161
zulu@1.7.154
zulu@1.7.141
zulu@1.7.131
zulu@1.7.101
zulu@1.7.95
> jabba install adopt@1.11.0-0
Downloading adopt@1.11.0-0 (https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11%2B28/OpenJDK11-jdk_x64_mac_hotspot_11_28.tar.gz)
189890053/189890053
Extracting /var/folders/x4/jg66qz6n3dqd4fwxgfsp3j2h0000gn/T/jabba-d-831976349 to /Users/pmonks/.jabba/jdk/adopt@1.11.0-0
/Users/pmonks/.jabba/jdk/adopt@1.11.0-0/Contents/Home/bin/java wasn't found. If you believe this is an error - please create a ticket at https://github.com/shyiko/jabba/issues (specify OS and command that was used)
>
```

Install OpenJDK v11, just to ensure Jabba is functional:

```
> jabba install openjdk@1.11.0
Downloading openjdk@1.11.0 (https://download.java.net/java/ga/jdk11/openjdk-11_osx-x64_bin.tar.gz)
182717049/182717049
Extracting /var/folders/x4/jg66qz6n3dqd4fwxgfsp3j2h0000gn/T/jabba-d-740395294 to /Users/pmonks/.jabba/jdk/openjdk@1.11.0
openjdk@1.11 -> /Users/pmonks/.jabba/jdk/openjdk@1.11.0
> jabba use openjdk@1.11.0
> java -version
openjdk version "11" 2018-09-25
OpenJDK Runtime Environment 18.9 (build 11+28)
OpenJDK 64-Bit Server VM 18.9 (build 11+28, mixed mode)
>
```

**Comment by shyiko** [comment]
Thank you, Peter. I'll have a new release shortly.

**Comment by shyiko** [comment]
Fixed in 0.11.0.
